src/Logary/Registry.fs

```diff
@@ -206,6 +214,59 @@ module Registry =
          && not ("true" = Env.varDefault "LOGARY_I_PROMISE_I_HAVE_PURCHASED_LICENSE" (fun () -> "false")) then
          failwith "You must purchase a license for Logary to run it on or with IIS or Kestrel."
 
+    // check if msg has enable metric counter
+    let hookupMetric (metricRegistry: MetricRegistry) (message: Message) =
+      // process log message as counter
+      let counterMetricConf = message |> Message.tryGetCounterMetricConf
+      match counterMetricConf with
+      | None -> do ()
+      | Some counterMetricConf ->
+        let counterMetric = GaugeConf.create counterMetricConf |> metricRegistry.registerMetric
+        let counter =
+          match counterMetricConf.labelNames with
+          | [||] -> counterMetric.noLabels
+          | labelNames ->
+            // find lable name's value from message's context, fields goes first
+            let labelValues =
+              labelNames |> Array.map (fun labelName ->
+                match message |> Message.tryGetField labelName with
+                | Some labelValue -> string labelValue
+                | None ->
+                  match message |> Message.tryGetContext labelName with
+                  | Some labelValue -> string labelValue
+                  | None -> failwithf "label name (%s) not found in message,maybe you should metric with nolables" labelName
```

Comment:
Do we want to throw from within Logary? :-s Also; spelling.

Reply:
since if pipe doesn't introduce any async/parall processing, this will fail at user's callsite, but given a warning message to its label value may be a good idea. let user find this issue when query metric at backend (like prometheus)